doc/security.md

```diff
@@ -23,7 +37,47 @@ certificate for any client-server communication.
 To cause certificates to be regenerated, simply remove the old ones. On the
 next connection a new certificate will be generated.
 
-## Adding a remote with a default setup
+## Role Based Access Control (RBAC)
+LXD supports integrating with the Canonical RBAC service.
+
+This uses Candid based authentication with the RBAC service maintaining
+roles to user/group relationships. Roles can be assigned to individual
+projects, to all projects or to the entire LXD instance.
```

Comment:
@stgraber in the case of assigning a role to "all projects", what doesn't that cover compared to an entire LXD instance?

Reply:
Entire LXD instance covers anything which isn't part of a project, including things like network/storage/server configuration.